tests/src/test/scala/scalaz/TheseTTest.scala

```diff
@@ -0,0 +1,26 @@
+package scalaz
+
+import scalaz.scalacheck.ScalazProperties._
+import scalaz.scalacheck.ScalazArbitrary._
+import std.AllInstances._
+import org.scalacheck.Prop.forAll
+
+
+object TheseTTest extends SpecLite {
+  type TheseTList[A, B] = TheseT[List, A, B]
+  type TheseTListInt[A] = TheseT[List, Int, A]
+  type TheseTOptionInt[A] = TheseT[Option, Int, A]
+
+  implicit def TheseTListIntEqual[A: Equal]: Equal[TheseTListInt[A]] = new Equal[TheseTListInt[A]] {
```

Comment:
I think this is unnecessary because now there is `implicit def theseTEqual` in `ThereT.scala`.